packages/react-components/.storybook/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
Is this basically just to check the JS files in this folder? (Also anything we need to do to avoid checking JS in any node_modules?)

Reply:
_context/note:_
-  this is present in every vNext package ( new DX stage 1 )

>  to check the JS files in this folder
- yes this is needed to enable `checkJs` to check files within `./storybook` and to provide best possible DX via intelisense

> Also anything we need to do to avoid checking JS in any node_modules

good thinking! fortunately for us node modules are ignored so no need :)
![2021-08-13 at 10 49](https://user-images.githubusercontent.com/1223799/129331174-9215f909-509d-47a6-98ef-73cfadb30c6d.png)